inspire/base/templates/accounts/settings/profile_base.html

```diff
@@ -0,0 +1,32 @@
+{#
+### This file is part of INSPIRE.
+## Copyright (C) 2014, 2015 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{%- import "accounts/settings/helpers.html" as helpers with context %}
+{%- from "_formhelpers.html" import render_field with context %}
+{%- extends "accounts/settings/index_base.html" -%}
+
+{% block settings_body %}
+{{helpers.panel_start('Profile', icon='fa fa-user fa-fw')}}
```

Comment:
To avoid future and likely confusion, we should, from day 1, make the distinction clear between user accounts and author profiles. I guess this page concerns users, so it should be renamed to something like "User account" or "User details".